src/controller/controlador_clientes.py

```diff
@@ -25,10 +25,15 @@ def abre_tela(self):
         while True:
             lista_opcoes[self.__tela_clientes.menu(list(lista_opcoes.keys()))]()
 
-    def cadastrar_cliente(self) -> Cliente:
+    def cadastrar_cliente(self):
```

Comment:
Por que você parou de retornar o cliente?
```suggestion
    def cadastrar_cliente(self -> Cliente):
```